templates/_base.html

```diff
@@ -59,19 +59,12 @@
   </style>
 
   {% block css %}{% endblock %}
-  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
-
-  <script type="module">
-    import 'https://unpkg.com/@polymer/paper-styles/color.js?module';
-    import '/static/elements/icons.js';
-
-    import 'https://unpkg.com/@polymer/app-layout/app-drawer-layout/app-drawer-layout.js?module';
-    import 'https://unpkg.com/@polymer/app-layout/app-header-layout/app-header-layout.js?module';
-    import 'https://unpkg.com/@polymer/app-layout/app-scroll-effects/app-scroll-effects.js?module';
-    import 'https://unpkg.com/@polymer/app-layout/app-drawer/app-drawer.js?module';
-    import 'https://unpkg.com/@polymer/app-layout/app-header/app-header.js?module';
-    import '/static/elements/chromedash-toast.js';
-  </script>
+  
+  {# Metric is need by sw registration and page code. #}
+  <script>{% inline_file "/static/js/metric.min.js" %}</script>
+
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2/webcomponents-loader.js"></script>
```

Comment:
I think it's fine to load it synchronously for now, but it does mean it will use `document.write()` to add the polyfills.

Might be good to switch to async loading in the future, https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs#asynchronous.

Reply:
Updated.